regression-test/data/nereids_function_p0/fn_test_ip.csv

```diff
@@ -0,0 +1,100 @@
+1;127.0.0.1;::1
```

Comment:
You can change file name from fn_test_ip.csv to test_ip.csv, since the test data can be used not only by function tests but also by all ip tests.